src/Microsoft.ML.TensorFlow/TensorFlow/TensorFlowModelContext.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+
+namespace Microsoft.ML.Transforms.TensorFlow
+{
+    /// <summary>
+    /// This class holds the information related to TensorFLow model and session.
+    /// It provides a convenient way to query model schema
+    ///     - Get complete schema by calling <see cref="GetModelSchema()"/>
+    ///     - Get schema related to model input by calling <see cref="GetInputSchema()"/>
+    /// </summary>
+    public class TensorFlowModelContext
```

Comment:
>TensorFlowModelContext [](start = 17, length = 22)

can we think of a better name than `Context` ? We are trying to limit ML.NET to just one context, called `MLContext`

Reply:
Maybe something along the lines of 'handle', or I don't know, `Info` ?

---
In reply to: [223784755](https://github.com/dotnet/machinelearning/pull/1191#discussion_r223784755) [](ancestors = 223784755)

Reply:
`TensorFlowModel` sounds much better I think

---
In reply to: [223785002](https://github.com/dotnet/machinelearning/pull/1191#discussion_r223785002) [](ancestors = 223785002,223784755)